src/_routing/_components/RootStackNavigator.tsx

```diff
@@ -106,7 +111,15 @@ export const RootStackNavigator = () => {
             name="RedeemedReward"
             options={({ navigation, route }) => ({
               headerBackTitle: '',
-              headerRight: props => route.params?.isModal ? <HeaderBackButton {...props} backImage={() => <Icon color="neutral.0" name="Close" size={14} />} onPress={() => navigation.popToTop()} /> : null,
+              headerRight: props =>
+                route.params?.isModal ? (
+                  <HeaderBackButton
+                    {...props}
+                    backImage={() => <Icon color="neutral.0" name="Close" size={14} />}
+                    onPress={() => navigation.pop()}
+                    style={{ marginRight: -10, padding: 10 }}
```

Comment:
🤮 

(als ge uw pressArea wilt vergroten kunt ge ook altijd uw hitSlop aanpassen he :D Als het iets met visuele feedback is dan is het door de vingers te zien)